src/pages/content/word-list.mdx

```diff
@@ -1,12 +1,707 @@
+
+This noun describes a UI element that lets users pick from a series of options.
+Always hyphenate it.
+
+### for example, e.g.
+
+Keep it conversational by using for example. The scholarly way to say “for
+example” is by using “e.g.,” which stands for exempli gratia.
+
+### either
+
+Use either only for a decision between two choices.
+
+<BestPractice>
+  <Do>
+    <p>You may choose either bacon or tofu.</p>
+  </Do>
+  <Dont>
+    <p>You may choose either bacon, snake meat, or tofu.</p>
+  </Dont>
+</BestPractice>
+
+### email, emails
+
+No hyphen needed.
+
+### feedback
+
+Use give feedback or for some Early Access Program (EAP) links, share your
+thoughts.
+
+<BestPractice>
+  <Do>
+    <p>Give feedback</p>
+  </Do>
+  <Dont>
+    <p>Provide feedback</p>
+  </Dont>
+</BestPractice>
+
+### full and fully
+
+Avoid full and fully unless you’re writing marketing copy. Even then, only use
+to mean complete.
+
+<BestPractice>
+  <Do>
+    <ul>
+      <li>Full-featured agent workflow</li>
+      <li>Full ticket history</li>
+    </ul>
+  </Do>
+  <Dont>
+    <ul>
+      <li>Fully customizable</li>
+      <li>Full live chat software functionality</li>
+    </ul>
+  </Dont>
+</BestPractice>
+
+### greater, better
+
+Avoid using subjective terms when talking about system requirements or versions.
+Use objective terms instead.
+
+<BestPractice>
+  <Do>
+    <ul>
+      <li>Update to the latest version of Google Chrome</li>
+      <li>Chat requires Microsoft Edge v.14 or higher</li>
+    </ul>
+  </Do>
+  <Dont>
+    <ul>
+      <li>Update to the best version of Explore</li>
+      <li>Chat requires Microsoft Edge v.14 or better</li>
+    </ul>
+  </Dont>
+</BestPractice>
+
+### Help Center
+
+Capitalize both words. Also, refer to it as Help Center, not the Help Center.
+
+Example: "Edit articles in Help Center."
+
+A possessive pronoun is okay.
+
+Example: “Edit articles in your Help Center.”
+
+### help desk
+
+Zendesk makes customer service software, not help desk software. The term help
+desk should only be used when specifically addressing IT or help desk issues.
+
+In most cases, you can use Zendesk Support instead.
+
+Example: "You can use Zendesk Support to improve your relationship with your
+customers."
+
+### hint
+
+Don’t use it as a heading for a type of note. Use tip instead.
+
+<!-- markdownlint-disable -->
+
+### i.e.
+
+Abbreviation for id est, meaning “that is.” Use that is or in other words
+instead.
+
+### incoming, outgoing
+
+Use to refer to email messages that are being downloaded or being sent. Do not
+use inbound and outbound unless you have no other choice.
+
+### Insights
+
+Capitalize when referring to the Zendesk product.
+
+### JavaScript
+
+Use [Pascal case](https://en.wikipedia.org/wiki/Camel_case). Not Javascript or javascript.
+
+### knowledge base
+
+Two words, lowercase. Sometimes called KB for short.
+
+### Learn about vs. learn more
+
+We reserve the linked words "Learn about" for links to the Zendesk Help Center.
```

Comment:
We reserve the linked words "Learn about" for links to the _Zendesk Help Center_. Avoid the standalone "Learn more" to prevent the repetition of the phrase for screen readers. Include detail about what the user will learn.